src/DataProtection/DataProtection/src/DataProtectionServiceCollectionExtensions.cs

```diff
@@ -34,6 +34,7 @@ public static IDataProtectionBuilder AddDataProtection(this IServiceCollection s
         ArgumentNullThrowHelper.ThrowIfNull(services);
 
         services.TryAddSingleton<IActivator, TypeForwardingActivator>();
+        services.TryAddEnumerable(ServiceDescriptor.Singleton<ILoggerProvider, MetricsLoggerProvider>());
```

Comment:
AFAICT, I shouldn't register a meter factory - someone else will do that.  There are reasons to use Data Protection without one.

Reply:
Thats correct. The AddMetrics() services extension method is responsible for the factory, the same way AddLogging() handles the ILoggerFactory.